Beistellung_Implementierung/system/Coding/src/app/State/SetParameters.h

```diff
@@ -0,0 +1,50 @@
+/***************************************************************************************************
+  (c) NewTec GmbH 2024   -   www.newtec.de
+***************************************************************************************************/
+/**
+ * @file       SetParameters.h
+ *
+ *    Module handles setting of parametersets and returns the currently selected parameter set on request
+ */
+/**************************************************************************************************/
+#ifndef SETPARAMETERS_H
+#define SETPARAMETERS_H
+#include "Common/Types.h"
```

Comment:
include in the wrong section

Reply:
(We already made a lot of changes to the code of the implementation phase for a running final version. Therefore changes won't be adresses directly in this old version but in the new one.)

This was already fixed in the new version.